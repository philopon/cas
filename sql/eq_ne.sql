SELECT '24311-42-1'::cas = '24311-42-1'::cas;
SELECT '24311-42-1'::cas != '24311-42-1'::cas;

SELECT '24311-42-1'::cas = '34311-42-1'::cas;
SELECT '24311-42-1'::cas != '34311-42-1'::cas;

SELECT '24311-42-1'::cas = '24311-46-1'::cas;
SELECT '24311-42-1'::cas != '24311-46-1'::cas;

SELECT '24311-42-3'::cas = '24311-42-1'::cas;
SELECT '24311-42-3'::cas != '24311-42-1'::cas;

SELECT '24311-62-3'::cas = '24311-42-1'::cas;
SELECT '24311-62-3'::cas != '24311-42-1'::cas;

SELECT '21311-62-3'::cas = '24311-42-1'::cas;
SELECT '21311-62-3'::cas != '24311-42-1'::cas;
