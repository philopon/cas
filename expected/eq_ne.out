SELECT '24311-42-1'::cas = '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-42-1'::cas != '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-1'::cas = '34311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-1'::cas != '34311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-42-1'::cas = '24311-46-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-1'::cas != '24311-46-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-42-3'::cas = '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-3'::cas != '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-62-3'::cas = '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-62-3'::cas != '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '21311-62-3'::cas = '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '21311-62-3'::cas != '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

